app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -142,4 +138,20 @@
             android:textStyle="bold" />
     </LinearLayout>
 
+    <FrameLayout
```

Comment:
Why FlameLayout is used and also Fragments need to be in a seperate simple activity and seperate fragments layout